src/ellipsoid.jl

```diff
@@ -67,12 +67,7 @@ function boundpts(b::Ellipsoid{N}) where {N}
     # n is (n .* r2) / sqrt(r2' * n.^2).  Below is the broadcasted version of this over a
     # matrix n, whose each column is a direction normal.  Once calculated, we need to
     # change the coordinates back to the original coordinates.
-    #
-    # This operation can be written M = b.p' * ((ndir .* r2) ./ sqrt.(r2' * ndir.^2)), but
-    # the resulting M is not an SMatrix.  (The calculation involves broadcasted division by
-    # a row vector, which leads to a non-SMatrix.)  Therefore, we first calculate M'
-    # (which remains SMatrix) and recover M.
-    M = (((ndir .* r2)' ./ sqrt.(ndir'.^2 * r2)) * b.p)'
+    M = b.p' * ((ndir .* r2) ./ sqrt.(r2' * ndir.^2))
```

Comment:
Isn't `r2` a real scalar, and hence doesn't need the `'`?   Also use `.*` to fuse that multiply with the other dot ops.

Reply:
No, in this case `r2` is a vector of the radii of the ellipsoid, and `ndir` is a matrix, so I think the operations are used correctly.

Reply:
Ah, I see.